roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -126,6 +126,12 @@ spec:
         - image: '{{ _redis_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           name: redis
+ {% if redis_command %}
+          command: {{ redis_command }}
+{% endif %}
+{% if redis_args %}
+          args: {{ redis_args }}
```

Comment:
Same here, this already exists, currently, your templaated redis_args wouldn't be used at all, the entry further down on line 140 would be used instead ([here](https://github.com/ansible/awx-operator/pull/1427/files#diff-a4af5f6d20ffe00a9f00fda1663b933b856c4a57435a796c4bad888fef83730dR140)).